vueui/src/components/view/medical/bodyscanner.vue

```diff
@@ -73,6 +73,7 @@
           <vui-group>
             <vui-group-item label="BP:" :style="{color:getPressureClass(blood_pressure_level)}"> {{blood_pressure}} </vui-group-item>
             <vui-group-item label="Blood Oxygenation:"> {{ Math.round(blood_o2) }}% </vui-group-item>
+            <vui-group-item label="Blood Volume:"> {{ Math.round(blood_volume) }}% </vui-group-item>
```

Comment:
/mob/living/carbon/human/proc/get_blood_volume() already rounds this value to the nearest 1%

Not really worth the effort to change, just wanted to be pedantic.